### PR TITLE
Apr 2216 update confirm withdrawal content

### PR DESCRIPTION
**Pull request by ConorTill**
Contains work for APR-2216, -2217 and -2218

### CODE REVIEW

File: src/SFA.DAS.RoatpGateway.Web/Services/GatewayApplicationActionsOrchestrator.cs [hunk 1]
```diff
@@ -22,17 +22,20 @@ public async Task<RoatpWithdrawApplicationViewModel> GetWithdrawApplicationViewM
         {
             RoatpWithdrawApplicationViewModel viewModel = null;
 
-            var commonDetails = await _applyApiClient.GetPageCommonDetails(applicationId, _commonDetailsPageId, userName);
-            if (commonDetails != null)
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Services/GatewayApplicationActionsOrchestrator.cs:26]: was this change from commonDetails intentional? Maybe @MarkFCain can advise as I thought it followed some sorta template

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcomeRejected.cshtml [hunk 2]
```diff
@@ -46,12 +47,87 @@
                             <h1 class="govuk-fieldset__heading">Are you sure you want to reject this application?</h1>
                         </legend>
 
+                        <dl class="govuk-summary-list">
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmOutcomeRejected.cshtml:50]: This is screaming to be in a partial view. Do we have one already?

File: src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmWithdrawApplication.cshtml [hunk 3]
```diff
@@ -52,22 +52,76 @@
...
-                            </p>
-                            <p class="govuk-body govuk-!-margin-bottom-0">
-                                Provider route: @Model.ApplicationRoute
-                            </p>
-                            <p class="govuk-body govuk-!-margin-bottom-0">
-                                Date application submitted: @Model.ApplicationSubmittedOn.ToSfaShortDateString()
-                            </p>
-                        </div>
+                        <dl class="govuk-summary-list">
+                            @if (!string.IsNullOrWhiteSpace(Model.Ukprn))
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpGateway.Web/Views/Gateway/ConfirmWithdrawApplication.cshtml:58]: partial view as mentioned above

File: src/SFA.DAS.RoatpGateway.Web/Services/GatewayApplicationActionsOrchestrator.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.RoatpGateway.Web/Services/GatewayApplicationActionsOrchestrator.cs:26]: It looks like this page needs stuff that isn't in common details (like referenceNumber, contact email), so these changes are fine imo.  You need to go get application and contact details anyway, so no need to do another call to common details here.

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**